modules/hmmcopy/gccounter/main.nf

```diff
@@ -0,0 +1,40 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+def VERSION = '0.99.0'
```

Comment:
This version does not match the conda/container versions

Reply:
Hmm, I could have sworn I had seen a 0.99 version number on the github link, but can't see it now. The tool itself doesn't have any version information that I can find. I'll just use the 0.1.1 now then.